apps/site/src/views/Home/sections/FAQ/FAQ.module.scss

```diff
@@ -0,0 +1,121 @@
+@use "zothacks-theme" as theme;
+@use "bootstrap-utils" as utils;
+
+.faq {
+	position: relative;
+	display: flex;
+	flex-direction: column;
+
+	margin-left: 40px;
+	margin-right: 40px;
+	padding-bottom: 30px;
+
+	background-color: theme.$beige;
+}
+
+.title {
+	align-self: center;
+	margin-top: 24px;
+	margin-bottom: 24px;
```

Comment:
Nitpick/chore: using `rem` would be more appropriate for these margins.

Reply:
fixed